### PR TITLE
PHP code snippets for Rackspace CDN getting started guide

### PR DESCRIPTION
**Pull request by ycombinator**
Also, fixing placeholders in curl code snippets.

Attn: @jamiehannaford, @rdodev

### CODE REVIEW

**Comment by rdodev** [comment]
LGTM :shipit:

File: src/docs/cdn/getting-started/samples/create_service.rst [hunk 1]
```diff
@@ -16,7 +16,20 @@
 
 .. code-block:: php
 
-  // Not currently supported by this SDK
+    $service = $cdnService->createService(array(
+        'name'     => 'example_site',
```
Comment (jamiehannaford) [inline src/docs/cdn/getting-started/samples/create_service.rst]: I don't really mind, but for the sake of consistency what do you think about using placeholders?

File: src/docs/cdn/getting-started/samples/update_service.rst [hunk 2]
```diff
@@ -16,7 +16,15 @@
 
 .. code-block:: php
 
-  // Not currently supported by this SDK
+    $service->update(array(
+        'origins' => array(
+            array(
+                'origin' => '44.33.22.11',
+                'port'   => 80,
+                'ssl'    => false
```
Comment (jamiehannaford) [inline src/docs/cdn/getting-started/samples/update_service.rst]: Same here

File: src/docs/cdn/getting-started/samples/create_service.rst (hunk 1 above)
Comment (ycombinator) [inline src/docs/cdn/getting-started/samples/create_service.rst]: Sure, I was just following what we are doing in other guides for names of resources. For example: https://developer.rackspace.com/docs/cloud-servers/getting-started/#build-server, and https://developer.rackspace.com/docs/cloud-queues/getting-started/#use-the-api.

Comment (ycombinator) [inline src/docs/cdn/getting-started/samples/create_service.rst]: I guess the question is what determines when to use a placeholder vs. when to use sample data? @rdodev, any thoughts?

**Comment by rdodev** [comment]
@ycombinator expressiveness vs. consistency via placeholders is a fine line to toe. And to be honest, the argument can go either way. I don't have a definite answer, sorry :(

File: src/docs/cdn/getting-started/samples/create_service.rst (hunk 1 above)
Comment (ycombinator) [inline src/docs/cdn/getting-started/samples/create_service.rst]: @jamiehannaford: See [Ruben's response](#issuecomment-71194193). I think, in this case, the sample data is more expressive of what it represents &mdash; a human-readable identifier for the resource being created &mdash; than a placeholder might be. But this is a subjective call and I am biased as author :) What do you think?

**Comment by jamiehannaford** [comment]
@ycombinator @rdodev Based on what Shaunak said, I have no problem with the sample data - I agree it's more expressive than placeholders here. LGTM :+1:

**Comment by ycombinator** [comment]
I see @rdodev's :shipit: and @jamiehannaford's :+1: and I raise them a merge.

**Pull request merged**